code/game/dna/dna2.dm

```diff
@@ -61,7 +61,9 @@ var/global/list/assigned_blocks[DNA_SE_LENGTH]
 	// DO NOT FUCK WITH THESE OR BYOND WILL EAT YOUR FACE
 	var/uni_identity="" // Encoded UI
 	var/struc_enzymes="" // Encoded SE
-	var/unique_enzymes="" // MD5 of player name
+	var/unique_enzymes="" // MD5 of player genetic marker value
```

Comment:
```suggestion
	var/unique_enzymes="" // MD5 of character genetic marker value
```
Always bugged me how older code doesn't maintain a proper distinction between player and character. Not blocking, but I'd prefer it changed if you're doing any other changes.

Reply:
See the suggested change in the comment. It's a nitpick so I'm not too pressed about it, but if you make additional changes to the current state of the PR I'd like it to be done.